### PR TITLE
add table of packages built by the community

### PR DESCRIPTION
**Pull request by kalebskeithley**
covers 3.6 through 3.9 inclusive

Signed-off-by: Kaleb S. KEITHLEY <kkeithle@redhat.com>

### CODE REVIEW

File: Install-Guide/Community_Packages.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+### Community Packages
+
+An **X** means packages are planned to be in the repository.
```
Comment (prashanthpai) [inline Install-Guide/Community_Packages.md]: This (line 3 to 6) is rendered as a single line. You could either convert these to a markdown list or add two or more whitespaces at the end which will be rendered as linebreak.

File: Install-Guide/Community_Packages.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+A **—** means we have no plans to build the version for the repository.
+**d.g.o** means packages will (also) be provided on https://download.gluster.org
+**DNF/YUM** means the packages are included in the Fedora updates or updates-testing repos.
+
+|                   |             |   3.9    |   3.8    |   3.7    |   3.6    |
+|-------------------|-------------|:--------:|:--------:|:--------:|:--------:|
+|CentOS Storage SIG¹|el5          |    —     |    —     |   d.g.o  |   d.g.o  |
```
Comment (prashanthpai) [inline Install-Guide/Community_Packages.md]: This numerical superscript here (`1`) and elsewhere. What do they point to or signify ?

**Comment by kalebskeithley** [comment]
I pushed an updated version

File: Install-Guide/Community_Packages.md [hunk 3]
```diff
@@ -34,3 +34,11 @@ A **—** means we have no plans to build the version for the repository.
 |                   |Leap 42.X    |    X     |    X     |    X     |    —     |
 |                   |SLES11       |    —     |    —     |    —     |    X     |
 |                   |SLES12       |    X     |    X     |    X     |    X     |
+
+
+	SLES12 	X
+	X 	X 	X
+
+¹ https://wiki.centos.org/SpecialInterestGroup/Storage
```
Comment (prashanthpai) [inline Install-Guide/Community_Packages.md]: linebreaks here too.

File: Install-Guide/Community_Packages.md [hunk 4]
```diff
@@ -34,3 +34,11 @@ A **—** means we have no plans to build the version for the repository.
 |                   |Leap 42.X    |    X     |    X     |    X     |    —     |
 |                   |SLES11       |    —     |    —     |    —     |    X     |
 |                   |SLES12       |    X     |    X     |    X     |    X     |
+
+
+	SLES12 	X
```
Comment (prashanthpai) [inline Install-Guide/Community_Packages.md]: There is something wrong here. You could use any online markdown editor to check rendering.

**Comment by kalebskeithley** [comment]
done

File: Install-Guide/Community_Packages.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+|                   |Jessie (8)   |  d.g.o   |  d.g.o   |  d.g.o   |  d.g.o   |
+|                   |Stretch (9)  |  d.g.o   |  d.g.o   |  d.g.o   |  d.g.o   |
+|                   |             |          |          |          |          |
+| SuSE Build System³|OpenSuSE13   |    X     |    X     |    X     |    X     |
+|                   |Leap 42.X    |    X     |    X     |    X     |    —     |
+|                   |SLES11       |    —     |    —     |    —     |    X     |
+|                   |SLES12       |    X     |    X     |    X     |    X     |
+
+
+	SLES12 	X
```
Comment (prashanthpai) [inline Install-Guide/Community_Packages.md]: These lines 39-40 must be a mistake ?

Check the rendering here: https://github.com/kalebskeithley/glusterdocs/blob/f2eb59f1f1311638a631831cffba600ea30bccbb/Install-Guide/Community_Packages.md

**Pull request merged**